packages/components/src/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,20 @@
+:root {
+  --progress-bar--borderRadius: 1rem;
+  --progress-bar--height: 1rem;
+  --progress-bar--backgroundColor: var(--color-grey--lightest);
+  --progress-bar--barColor: var(--color-lime);
+}
+
+.container {
+  width: 100%;
+  height: var(--progress-bar--height);
+  border-radius: var(--progress-bar--borderRadius);
+  background: var(--progress-bar--backgroundColor);
+}
+
+.content {
+  height: var(--progress-bar--height);
+  border-radius: var(--progress-bar--borderRadius);
+  background-color: var(--progress-bar--barColor);
+  transition: width 0.75s ease-out;
```

Comment:
we have a `--timing-slow` css variable available at `:root`. Would we want to consider this here instead of a custom transition time?